scripts/stanford-filter.script

```diff
@@ -108,6 +108,64 @@
     +(ManufacturerModelName.containsIgnoreCase("Sensation"))
    )
   )
+
+   // Siemens SOMATOM Definition Edge, SOMATOM DEFINITION AS+, SOMATOM FORCE
+  +(Manufacturer.startsWithIgnoreCase("SIEMENS") * (Modality.equalsIgnoreCase("CT") + Modality.equals("PT"))
+   *(ImageType.contains("DERIVED"))
+   *
+   (
+    (
+        (ManufacturerModelName.containsIgnoreCase("SOMATOM Definition Edge") + ManufacturerModelName.containsIgnoreCase("SOMATOM FORCE")) 
+         * [0018,1020].containsIgnoreCase("syngo CT VB20A")
+    )
+    +
+    (
+        ManufacturerModelName.containsIgnoreCase("SOMATOM DEFINITION AS+")
+         * ([0018,1020].containsIgnoreCase("syngo CT VB20A") + [0018,1020].containsIgnoreCase("syngo CT VA48A"))
+    )
+   )
+   //*(
+   //   (Rows.equals("679") * Columns.equals("512"))
+   //   +(Rows.equals("512") * (Columns.equals("512") + Columns.equals("547") + Columns.equals("605") + Columns.equals("630") + Columns.equals("632") + Columns.equals("682") + Columns.equals("708")))
+   //   +(Rows.equals("665") * Columns.equals("512"))
+   //)
+   )
+
+   // Siemens SOMATOM DEFINITION FLASH, version SYNGO CT VB20A
+  +(Manufacturer.startsWithIgnoreCase("SIEMENS") * (Modality.equalsIgnoreCase("CT") + Modality.equals("PT"))
+   *(ImageType.contains("DERIVED"))
+   *ManufacturerModelName.containsIgnoreCase("SOMATOM DEFINITION FLASH") 
+   *[0018,1020].containsIgnoreCase("syngo CT VB20A")
+   *(
+    [0018,1018].containsIgnoreCase("Volume Viewer")
+    +
+    [0018,1018].containsIgnoreCase("")
+    )
+   )
+
+   // Siemens secondary_capture_device_manufacturer AQNET
+  +(Manufacturer.startsWithIgnoreCase("SIEMENS") * (Modality.equalsIgnoreCase("CT") + Modality.equals("PT"))
```

Comment:
Is this rule too broad? Let's check with Joe

Reply:
If these contain PHI, then you need row/cols values also. You won't know if it's too broad until you run the regression suite and see if there were unexpected changes for other DICOM files. It would be easier to read also if you indented the boolean logic. 
Also I'm paranoid and would add additional parens when + is involved:

+(Manufacturer.startsWithIgnoreCase("SIEMENS") * (Modality.equalsIgnoreCase("CT") + Modality.equals("PT"))
→ 
+(Manufacturer.startsWithIgnoreCase("SIEMENS") * ((Modality.equalsIgnoreCase("CT") + Modality.equals("PT")))